runelite-client/src/main/java/net/runelite/client/plugins/chatcommands/ChatCommandsPlugin.java

```diff
@@ -114,6 +118,8 @@
 	private String lastBossKill;
 	private int lastPb = -1;
 
+	private Map<HiscoreSkill, Integer> kcMap = new HashMap<>();
```

Comment:
What is the purpose of this?

Reply:
This stores the current KC for bosses when killed if they are a hiscore boss. Since the hiscore updates on logout and people like the real-time kc commands I'm using this map to store & send the actual kc. I figured since Jagex is hosting this KC data it's no longer necessary to persist it via the config. I believe the website is the only other place that uses this data and it could just query the hiscores as well.

Reply:
Based on discord feedback I've reverted the commit that removed config storage for boss kcs.